src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationProvider.cs

```diff
@@ -173,15 +179,159 @@ public async Task RefreshAsync(CancellationToken cancellationToken)
                         if (InitializationCacheExpires < DateTimeOffset.UtcNow)
                         {
                             InitializationCacheExpires = DateTimeOffset.UtcNow.Add(MinCacheExpirationInterval);
-                            await LoadAll(ignoreFailures: false, cancellationToken).ConfigureAwait(false);
+                            await LoadAllWithFailOverPolicyAsync(ignoreFailures: false, cancellationToken).ConfigureAwait(false);

```

Comment:
There are a lot of DateTimeOffset.UtcNow invocations, we could save a bit if we calculate it once where possible.

Reply:
Yes, that would save us a bit. I updated wherever we are calculating `UtcNow` frequently.